NExtends/Expressions/QueryConverterHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+
+namespace NExtends.Expressions
+{
+	public static class QueryConverterHelper
+    {
+        public static Expression<Func<TKeep, bool>> KeepExpressionsOfTypeInTree<TSource, TKeep>(Expression<Func<TSource, bool>> original, Dictionary<string, string> propertyRenaming, bool withReplacement)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Linq.Expressions.LambdaExpression' instead of 'System.Linq.Expressions.Expression<System.Func<TSource, bool>>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)